workers/unity/Assets/Gdk/Core/Error/ConnectionError.cs

```diff
@@ -0,0 +1,20 @@
+namespace Improbable.Gdk.Core
+{
+    public enum ConnectionErrorReseaon
+    {
+        CannotEstablishConnection,
+        DeploymentNotFound,
+        InvalidConfig
+    }
+
+    public struct ConnectionError : IErrorContext
```

Comment:
just checked out your branch and it can't find `IErrorContext`. did you forget to add a file?

Reply:
Yup :/ Forgot to add all new files. Added.